docs/DeveloperGuide.md

```diff
@@ -856,12 +883,81 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given name is invalid.
+* 3a. The given index is invalid.
 
     * 3a1. LinkedOUT shows an error message.
 
       Use case resumes at step 2.
 
+[Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
+
+---
+
+**Use case: Edit an applicant**
+
+**Preconditions: LinkedOUT application is launched**
+
+**Guarantees: Applicant will be edited only if the user input
+does not have any formatting issues.**
+
+**MSS**
+
+1.  User requests to list applicants.
+2.  LinkedOUT shows a list of applicants.
+3.  User requests to edit a specific applicant in the list.
+4.  LinkedOUT edits the applicant.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. LinkedOUT shows an error message.
+
+      Use case resumes at step 2.
+
+[Back to top <img src="images/back-to-top-icon.png" width="25px" />](#table-of-contents)
+
+---
+
+**Use case: Sort the list of applicants**
+
+**Preconditions: LinkedOUT application is launched**
+
+**Guarantees: A temporarily sorted list of applicants will be displayed
+only if the user input does not have any formatting issues.**
+
+**MSS**
+
+1.  User requests to list applicants.
+2.  LinkedOUT shows a list of applicants.
+3.  User requests to sort the list of applicants.
+4.  LinkedOUT shows the list of applicants in a sorted order.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. User input has invalid formatting.
+
+    * 3a1. LinkedOUT shows an error message, indicating formatting issues
+      as well as an example of correct input format.
```

Comment:
I think need to standardize whether to mention "indicating formatting issue as well as an example of correct input format" with other commands (like view add..). Let me know if you decide to include it or remove it so that I can update my part if u want to include it